plugins/woocommerce/includes/class-wc-order.php

```diff
@@ -14,6 +14,91 @@
  * These are regular WooCommerce orders, which extend the abstract order class.
  */
 class WC_Order extends WC_Abstract_Order {
+	/**
+	 * The following constants are used to define the order status.
+	 * For a full documentation on the public order statuses, please refer to the following link:
+	 * https://woocommerce.com/document/managing-orders/order-statuses/
+	 */
+
+	/**
+	 * The order has been received, but no payment has been made.
+	 *
+	 * @var string
+	 */
+	const STATUS_PENDING = 'pending';
```

Comment:
My preference would be to relocate the new constants into a new enum-alike class somewhere under `src` directory. WDYT?

Reply:
That was actually my initial idea, but I could not find any example in our current code. I just did it now [here](https://github.com/woocommerce/woocommerce/pull/51919/commits/736531352129ee01d5e976a89b6905d3a6ed6adc). Let me know what you think.